docker/keycloak.dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG KC_VERSION=latest
```

Comment:
Rather than latest can we take this in from the registry? So the kc version would be whatever the default tag is for the base keyckoak image.

Reply:
It's just a placeholder. The workflow passes the version as a `--build-arg`.